### PR TITLE
Add avatar component by Docux

### PR DESCRIPTION
**Pull request by Juniors017**

### CODE REVIEW

File: docs/component-library/new/Simplecard/index.mdx [hunk 1]
```diff
@@ -39,29 +39,29 @@ Creating the file and adding the source code for the component.
...
-};
+}
 
 
-export default  Card;
+export default AvatarContainer ;
+  
 ```
 
 </TabItem>
```
Comment (homotechsual) [inline docs/component-library/new/Simplecard/index.mdx:67]: Why is this editing a file in `SimpleCard`?

File: src/components/Avatar/AvatarContainer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}) => {   
+  const avatarPosition = vertical ? 'avatar--vertical' : '';
+
+  return (
+    <div className={clsx(
+        "avatar", 
+        className, 
+        avatarPosition
+        )} 
+
```
Comment (homotechsual) [inline src/components/Avatar/AvatarContainer/index.tsx:25]:
```suggestion
```

File: src/components/Avatar/AvatarContainer/index.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  return (
+    <div className={clsx(
+        "avatar", 
+        className, 
+        avatarPosition
+        )} 
+        
+        style={style}>
+
```
Comment (homotechsual) [inline src/components/Avatar/AvatarContainer/index.tsx:27]:
```suggestion
```

File: src/components/Avatar/AvatarImage/index.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  const avatarImageSizeClass = avatarSize ? `avatar__photo--${avatarSize}` : '';
+  
+  const imgElement = (
+    <img
+      className={clsx(
+      "avatar__photo", 
+      className, 
+      avatarImageSizeClass
+      )}
+
```
Comment (homotechsual) [inline src/components/Avatar/AvatarImage/index.tsx:36]:
```suggestion
```

File: src/components/Avatar/AvatarImage/index.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  const generatedAvatarUrl = useBaseUrl(avatarImageUrl);
+  const avatarImageSizeClass = avatarSize ? `avatar__photo--${avatarSize}` : '';
+  
+  const imgElement = (
+    <img
+      className={clsx(
+      "avatar__photo", 
+      className, 
+      avatarImageSizeClass
+      )}
```
Comment (homotechsual) [inline src/components/Avatar/AvatarImage/index.tsx:35]:
```suggestion
      className={clsx(
          "avatar__photo", 
          className, 
          avatarImageSizeClass
      )}
```

File: src/components/Avatar/AvatarImage/index.tsx [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+      style={style}
+      src={generatedAvatarUrl}
+      alt={alt}
+      title={title}
+
```
Comment (homotechsual) [inline src/components/Avatar/AvatarImage/index.tsx:41]:
```suggestion
```

**Review by homotechsual — CHANGES_REQUESTED** [review]

**Comment by Juniors017** [comment]
Hi @homotechsual 
I don t know why card componant was modificated.. it s old 😔
Sorry

**Review by Juniors017 — COMMENTED** [review]
Ok

**Comment by homotechsual** [comment]
Sorry but this now deletes the SimpleCard file rather than leaving it unchanged. This would break the SimpleCard demo/example.

**Pull request closed without merging**